openslides/mediafiles/urls.py

```diff
@@ -4,21 +4,6 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^$',
-        views.MediafileListView.as_view(),
-        name='mediafile_list'),
-
-    url(r'^new/$',
-        views.MediafileCreateView.as_view(),
-        name='mediafile_create'),
-
-    url(r'^(?P<pk>\d+)/edit/$',
-        views.MediafileUpdateView.as_view(),
-        name='mediafile_update'),
-
-    url(r'^(?P<pk>\d+)/del/$',
-        views.MediafileDeleteView.as_view(),
-        name='mediafile_delete'),
     url(r'^pdf/next/$', views.PdfNextView.as_view(), name='next_pdf_page'),
```

Comment:
Could you change the name of the following url-patterns to "mediafiles_*"? I know, this is not part of you pull request, but then we can close #1509